src/Form/ItemCheckbox.js

```diff
@@ -96,6 +101,11 @@ ItemCheckbox.propTypes = {
   // Optional value for setting the checked state of the checkbox,
   // should be either true for 'checked' or false 'unchecked'
   checked: React.PropTypes.bool,
+  // Optional value for setting disabled state
+  disabled: React.PropTypes.bool,
+  // Optional. Which field property is currently being edited
+  // (usually passed down from form definition)
+  editing: React.PropTypes.string,
   // Function to handle change event
   // (usually passed down from FieldCheckbox)
   handleChange: React.PropTypes.func,
```

Comment:
Oh wait nevermind, I suppose the event handler prop name is only standardized among the FieldTypes